velox/vector/LazyVector.cpp

```diff
@@ -41,7 +45,9 @@ void VectorLoader::load(
     vector_size_t resultSize,
     VectorPtr* result) {
   {
-    DeltaCpuWallTimer timer([&](auto& delta) { writeIOTiming(delta); });
+    DeltaCpuWallTimer lazyIoRecorder([&](auto& cpuDelta) {
```

Comment:
Can we have a new class for this as it now does delta cpu and io stats tracking. Maybe LazyIoStatsRecorder? We could just define in LazyVector.cpp? Thanks!